tests/drop_counters/test_drop_counters.py

```diff
@@ -415,10 +415,13 @@ def str_to_int(value):
 
 def verify_drop_counters(duthost, dut_iface, get_cnt_cli_cmd, column_key):
     """ Verify drop counter incremented on specific interface """
-    drops = get_pkt_drops(duthost, get_cnt_cli_cmd)[dut_iface][column_key]
-    drops = str_to_int(drops)
-
-    if drops != PKT_NUMBER:
+    for _ in range(3):
+        drops = get_pkt_drops(duthost, get_cnt_cli_cmd)[dut_iface][column_key]
+        drops = str_to_int(drops)
+        if drops == PKT_NUMBER:
+            break
+        time.sleep(1)
```

Comment:
Could you take a look at https://github.com/Azure/sonic-mgmt/blob/master/tests/common/utilities.py and use the `wait_until` method instead?

Reply:
Fixed